console/src/components/Services/Data/DataActions.js

```diff
@@ -264,10 +266,13 @@ const loadSchema = configOptions => {
         fetchTrackedTableListQuery(configOptions), // v1/query
         fetchTrackedTableFkQuery(configOptions),
         fetchTrackedTableReferencedFkQuery(configOptions),
-        fetchTrackedTableRemoteRelationshipQuery(configOptions),
       ],
     };
 
+    if (featuresCompatibility.RemoteRelationShips) {
```

Comment:
Use the constant `FT_REMOTE_RELATIONSHIPS` exported from `src/helpers/versionUtils`.